src/lwc/rd2ElevateCreditCardForm/rd2ElevateCreditCardForm.css

```diff
@@ -0,0 +1,41 @@
+.payment-services-iframe {
+    width: 100%;
+    height: 4.7rem;
+    border: none;
+}
+
+.iframe-container {
+    position: relative;
+}
+
+.credit-card-container {
+    border: 1px solid #E2E1E0;
+    border-radius: 0.25rem;
+    min-height: 5rem;
+    box-shadow: 0 2px 2px 0 rgba(0, 0, 0, 0.1);
+}
+
+.credit-card-header {
+    background-color: #E2E1E0;
```

Comment:
What if the User changes the theme in their org? Normally that would cause other colors to shift throughout the app, but here the background color is this section would remain the same even in an alternate theme. Is there a SLDS style that can be used to define this or a similar color so it's not hardcoded/fixed?

Reply:
Excellent point! The change has been made to use [design tokens](https://developer.salesforce.com/docs/atlas.en-us.lightning.meta/lightning/tokens_standard_force_base.htm) and the fallback colors (in the case token rendering does not work).